### PR TITLE
Meltan!

### PR DESCRIPTION
**Pull request by amative1**
Framework for Meltan quest.

### CODE REVIEW

**Comment by amative1** [comment]
Blocked by #2065

**Comment by amative1** [comment]
Need to rebase, too many conflicts

File: src/scripts/quests/QuestLineHelper.ts [hunk 1]
```diff
@@ -1968,11 +1968,181 @@ class QuestLineHelper {
         App.game.quests.questLines().push(hoopaDayPikabluQuestLine);
     }
 
+    public static createMeltanQuestLine() {
```
Comment (Jaaslet) [inline src/scripts/quests/QuestLineHelper.ts:1971]: The questlines needs to be moved up to the alola questlines

**Review by Jaaslet — CHANGES_REQUESTED** [review]
Looks good. Have some comments:
- I have fixed some small errors
- The merge conflicts are too big for me to handle them on github. It's after Jessica moved pokemonList to moduels. After that, the indexes in the update functions needs to be moved up once
- The temp battles are missing images. Or you at least need to point it to the correct image, if they already exists as dungeon trainers or something

File: src/scripts/quests/QuestLineHelper.ts (hunk 1 above)
Comment (amative1) [reply src/scripts/quests/QuestLineHelper.ts:1972]: Merge conflicts resolved, eslint passing.

Comment (amative1) [reply src/scripts/quests/QuestLineHelper.ts]: Questlines moved into Alola section. Not sure what you mean by the indexes in the update functions. And idk if we have images for any of the temp battles yet (at the very least the two grunts, and maybe Faba? We should have the Leaders.)

Comment (amative1) [reply src/scripts/quests/QuestLineHelper.ts]: (ah, I see, completely missed those Update lines... not that I would've figured out what needed fixing)

**Review by Jaaslet — APPROVED** [review]
I am going to approve this now. We are still missing some credits (i think?), but we can do that in an other PR, when we figure it out. I just want this merged :)
Looks good. Thanks for the PR

**Pull request merged**